### PR TITLE
Add basic sails and sailing questions

### PR DESCRIPTION
**Pull request by Shlepanets**

### CODE REVIEW

File: resources/questions/basic_sails_and_sailing_3.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+image: aground.png
```
Comment (justrp) [inline resources/questions/basic_sails_and_sailing_3.yml:1]: why?

File: resources/questions/basic_sails_and_sailing_1.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+question: |
+  Бегучий такелаж (running rigging) – это совокупность канатов и шкотов, используемая для регулировки угла парусов к ветру
+options:
+  - Правда
+  - Неправда
+correct: Правда
```
Comment (justrp) [inline resources/questions/basic_sails_and_sailing_1.yml:6]: why no newline?

File: resources/questions/basic_sails_and_sailing_5.yml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+image: aground.png
```
Comment (justrp) [inline resources/questions/basic_sails_and_sailing_5.yml:1]: why?

File: resources/questions/basic_sails_and_sailing_1.yml (hunk 2 above)
Comment (justrp) [reply resources/questions/basic_sails_and_sailing_1.yml:6]: ok, you got a couple of q with newlines at EOF, and part - without.
why?)

File: resources/questions/basic_sails_and_sailing_3.yml (hunk 1 above)
Comment (Shlepanets) [reply resources/questions/basic_sails_and_sailing_3.yml:1]: No idea

File: resources/questions/basic_sails_and_sailing_5.yml (hunk 3 above)
Comment (Shlepanets) [reply resources/questions/basic_sails_and_sailing_5.yml:1]: It was late =/

File: resources/questions/basic_sails_and_sailing_1.yml (hunk 2 above)
Comment (Shlepanets) [reply resources/questions/basic_sails_and_sailing_1.yml:6]: No reason, just random)
Will be fixed to one style

File: resources/questions/basic_sails_and_sailing_3.yml (hunk 1 above)
Comment (Shlepanets) [reply resources/questions/basic_sails_and_sailing_3.yml]: removed

File: resources/questions/basic_sails_and_sailing_5.yml (hunk 3 above)
Comment (Shlepanets) [reply resources/questions/basic_sails_and_sailing_5.yml]: removed

File: resources/questions/basic_sails_and_sailing_1.yml (hunk 2 above)
Comment (Shlepanets) [reply resources/questions/basic_sails_and_sailing_1.yml]: fixed

**Pull request merged**